sigmf-spec.md

```diff
@@ -689,21 +692,60 @@ deciding which open license fits your needs best. Cornell University has also
 created [a guide](https://data.research.cornell.edu/content/intellectual-property#data-licensing) 
 to help you make these choices.
 
-## SigMF Compliance by Applications
+## SigMF Compliance
 
-In order to be `SigMF Compliant`, an application MUST meet the following
+The term 'SigMF Compliant' is used throughout this document, which can take on
+one of several contextually dependent meanings. In order for a schema,
+recording, or application to be 'SigMF Compliant', specific conditions MUST be
+met, outlined in the following sections. Provided the below criteria are met, an
+application or dataset can indicate that it is 'SigMF Compliant'.
+
+### SigMF Schema Compliance
+
+In order to be 'SigMF Compliant', a schema MUST meet the following requirements:
+
+1. Adheres to and supports the metadata file naming conventions, `objects`,
+   `namespaces`, and `names` specified by this document.
+1. MUST contain all REQUIRED fields with the correct datatype listed the `core`
+   namespace, and any namespace listed in the `extensions` array.
+1. MUST NOT contain fields that are not outlined in the `core` or a listed
+   `extensions` namespace.
+
+### SigMF Recording Compliance
```

Comment:
I think a Recording being compliant should explicitly state that the Schema must be compliant. We've already got a bullet for the Dataset